src/contentsharecontroller/ContentShareMediaStreamBroker.ts

```diff
@@ -23,6 +23,7 @@ export default class ContentShareMediaStreamBroker implements MediaStreamBroker
 
   async acquireAudioInputStream(): Promise<MediaStream> {
     if (this._mediaStream.getAudioTracks().length === 0) {
+      this.logger.info('Synthesizing fake audio stream');
```

Comment:
We use the term "empty audio."
https://github.com/aws/amazon-chime-sdk-js/blob/7d7d0cbebdfc1a11113001ca76f2b7eddf379d7a/src/devicecontroller/DefaultDeviceController.ts#L943-L945

How about the following message?
> Create an empty audio device for the content share